src/Storage/Storage.Management/Models/PSBlobInventoryPolicy.cs

```diff
@@ -27,23 +29,21 @@ public class PSBlobInventoryPolicy
         public PSBlobInventoryPolicy()
         { }
 
-        public PSBlobInventoryPolicy(BlobInventoryPolicy policy, string ResourceGroupName, string StorageAccountName)
+        public PSBlobInventoryPolicy(Track2.BlobInventoryPolicyResource policy, string resourceGroupName, string storageAccountName)
         {
-            this.ResourceGroupName = ResourceGroupName;
-            this.StorageAccountName = StorageAccountName;
+            this.ResourceGroupName = resourceGroupName;
+            this.StorageAccountName = storageAccountName;
             this.Id = policy.Id;
-            this.Name = policy.Name;
-            this.Type = policy.Type;
-            this.LastModifiedTime = policy.LastModifiedTime;
-            this.SystemData = policy.SystemData is null ? null : new PSSystemData(policy.SystemData);
+            this.Name = policy.Data.Name;
+            this.Type = policy.Data.ResourceType;
+            this.LastModifiedTime = policy.Data.LastModifiedOn;
+            this.SystemData = policy.Data.SystemData is null ? null : new PSSystemData(policy.Data.SystemData);
+            this.Enabled = policy.Data.Policy.Enabled;
 
-            this.Enabled = policy.Policy.Enabled;
-            this.Destination = policy.Policy.Destination;
```

Comment:
Might need to add a comments to add the destination which is on a later API version.

Reply:
Done